src/main/java/com/cleanroommc/groovyscript/gameobjects/GameObjectHandlers.java

```diff
@@ -205,4 +208,12 @@ public static Result<Material> parseBlockMaterial(String mainArg, Object... args
         Material material = materials.get(mainArg);
         return material == null ? Result.error() : Result.some(material);
     }
+
+    public static @NotNull Result<NBTTagCompound> parseNBT(String mainArg, Object... args) {
+        try {
+            return Result.some(JsonToNBT.getTagFromJson(mainArg));
+        } catch (NBTException e) {
+            return Result.error("unable to parse provided nbt string");
+        }
+    }
```

Comment:
I dont think this is a good idea. This requires the nbt to be a string, but its usually a map. I think it would be better to make a method in `GroovyHelper` (its static star imported in all scripts) with a mao argument. There are methods in grs somewhere to convert map to nbt.

Reply:
Sounds good, I'll give a crack at this implementation.

Reply:
Allowed for passing a groovy nbt map in: [07e9dd1](https://github.com/CleanroomMC/GroovyScript/pull/95/commits/07e9dd17fad81f2392401bc8a5854cbd0ed45462)